inc/db_mysql.php

```diff
@@ -1353,7 +1353,7 @@ function replace_query($table, $replacements=array(), $default_field="", $insert
 	 */
 	function drop_column($table, $column)
 	{
-		return $this->write_query("ALTER TABLE {$this->table_prefix}{$table} DROP {$column}");
+		return $this->write_query("ALTER TABLE {$this->table_prefix}{$table} DROP `{$column}`");
```

Comment:
I think it's worth checking if `$column` is already quoted for these methods to prevent duplicate quotes.

Reply:
Not in the core (adding custom quoting for such methods would be hacky, but these plugins could break - not sure if worth the overhead)

Reply:
Overhead of mitigating creative approaches, rather than performance; will add